src/dbnode/server/server.go

```diff
@@ -768,8 +784,14 @@ func Run(runOpts RunOptions) {
 	closeTimeout := serverGracefulCloseTimeout
 	select {
 	case <-closedCh:
+		if err := bsGauge.Close(); err != nil {
```

Comment:
Is this in the same function as when `Start()` is called? If so I think we *should* just be able to call

```golang
defer bsGauge.Close()
```

right after calling start and it won't run until the database is stopped anyways because the main function never returns until the database shuts down (due to blocking on the `xos.WaitForInterrupt()` call). If you look throughout that function you'll see a few other examples of using `defer` for cleanup after the database node shuts down

Reply:
Done